docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -50,10 +51,13 @@ You must specify:
 
 You can also specify:
 
-- ``<queries>``: the queries to run over your database. You can
-  list one or more individual query files, specify a directory that will be
-  searched recursively for query files, or name a query suite that defines a
-  particular set of queries. If omitted, the default query suite for the language
+- ``...<query-specifications>``: a list of queries to run over your database. This
+  is a list of arguments. Where each argument can be:
+  - a path to a query file
+  - a path to a directory containing query files
+  - a path to a query suite file
+  - the name of a CodeQL query pack
```

Comment:
The formatting seems to be off here (at least in the docs generated by the checks on the PR). You may need an empty line before and after these nested bullets.